### ISSUE TITLE
Fix Project ID in GitLab Issues Worker + Release Worker question

### ISSUE THREAD

**Issue by sgoggins** [chaoss/augur#1070]
**Description:**
@mrsaicharan1 and @pratikmishra356  : The Gitlab issues worker is using our internal project ID, not gitlabs.   Please examine the `gitlab-issues+releases-patch` branch

**How to reproduce:**
1. from augur base `make install-dev`
2. slack for db credentials if needed
3. you will see a log hitting this endpoint, which is not correct: 
```

2020-12-27 17:14:55,049,49ms [PID: 76935] workers.gitlab_issues_worker.52000 [INFO] OAuth initialized

**URL OF REPO FOR ISSUE GATHERING**
2020-12-27 17:14:55,049,49ms [PID: 76935] workers.gitlab_issues_worker.52000 [INFO] Popped off message: {'job_type': 'MAINTAIN', 'models': ['gitlab_issues'], 'display_name': 'gitlab_issues model for url: https://gitlab.com/gitlab-org/gitlab-foss', 'given': {'git_url': 'https://gitlab.com/gitlab-org/gitlab-foss'}}

**INTERNAL PROJECT ID 25430 in API CALL**
2020-12-27 17:14:55,538,538ms [PID: 76935] workers.gitlab_issues_worker.52000 [INFO] Hitting endpoint: https://gitlab.com/api/v4/projects/25430/issues?per_page=100&state=opened&with_labels_details=True&page=1...


**Expected behavior:**
Call the GitLab Project ID

**BRANCH: `gitlab-issues+releases-path`

**Comment by sgoggins** [comment]
@ivanayov : In this branch it also seems that the release worker either: 
1. Only gets releases for the current year OR
2. Only gets the last 10 releases. There's some code in the GraphQL query that leads me to believe its for the last 12 months, but the only GitHub repo in my test database is Augur so I can't be sure. There is an explicit "10" in the GraphQL query. Can you look at that?

**Comment by sgoggins** [comment]
@ivanayov : The code's actually in all branches through `dev`, but I am working on the question here: `gitlab-issues+releases-patch` branch.

**Comment by pratikmishra356** [comment]
@sgoggins I didn't worked on gitlab workers , I guess @abhinavbajpai2012 worked on this.

**Comment by abhinavbajpai2012** [comment]
I worked on Merge Request worker. @mrsaicharan1 should be the right person for this.

**Comment by sgoggins** [comment]
@abhinavbajpai2012 : Can you take a look at this one. Sorry for the confusion. Github names are hard to follow. :) 

@mrsaicharan1 : If you can scan it and see where @abhinavbajpai2012 would insert a query to GitLab to retrieve their ID, possibly borrowing from your worker, that would be very nice. :D

**Comment by sgoggins** [comment]
I created a database called `augur_gitlab_patch` on one of our servers. It has a github and gitlab repo in it ... tiny ... slack for credentials if you can work on it.  Thanks! @mrsaicharan1  @abhinavbajpai2012

**Comment by ivanayov** [comment]
@sgoggins I'll work on the release worker to fix it

**Comment by sgoggins** [comment]
@ivanayov : In this branch it also seems that the release worker either:

    Only gets releases for the current year OR
    Only gets the last 10 releases. There's some code in the GraphQL query that leads me to believe its for the last 10 releases, but the only GitHub repo in my test database is Augur so I can't be sure. There is an explicit "10" in the GraphQL query; but we happen to have had 10 releases in the last 12 months. Can you look at that?


Have you had a chance to look into this?

**Comment by sgoggins** [comment]
I believe this issue is fixed.
